reo/nested_outputs.py

```diff
@@ -947,6 +947,16 @@ def list_of_string(input):
                   "type": float,
                   "description": "Year one chiller thermal production",
                   "units": "TonHr"
+                },
+                "year_one_absorp_chl_electric_consumption_series_kw": {
+                  "type": list_of_float,
+                  "description": "Year one hourly time series of absorption chiller thermal consumption",
```

Comment:
edit "description" to "Year one hourly time series of absorption chiller *electric* consumption"

Reply:
@danolis thanks for the review. I corrected the description in [this commit](https://github.com/NREL/REopt_Lite_API/pull/158/commits/ca0df333322943317d39d04da14968c9e00d05af).